test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -318,23 +318,160 @@ class MultipleDisposalsTriageControllerSpec
+              journey.copy(
+                newReturnTriageAnswers = Left(answers.copy(wasAUKResident = Some(true)))
+              )
+            )
+            )
+          )(Right(()))
+        }
+
+        checkIsRedirect(
+          performAction("multipleDisposalsWereYouAUKResident" -> "true"),
+          routes.MultipleDisposalsTriageController.checkYourAnswers()
+        )
+      }
+
+      "user has not answered were uk resident section and " +
+        "redirect to dummy page when user selects no" in {
+        val answers = IncompleteMultipleDisposalsAnswers.empty.copy(
+          individualUserType = Some(Self),
+          numberOfProperties = Some(2)
+        )
+        val (session, journey) = sessionDataWithStartingNewDraftReturn(answers)
+
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+          mockStoreSession(
+            session.copy(journeyStatus = Some(
+              journey.copy(
+                newReturnTriageAnswers = Left(answers.copy(wasAUKResident = Some(false)))
+              )
+            )
+            )
+          )(Right(()))
+        }
+
+        checkIsRedirect(
+          performAction("multipleDisposalsWereYouAUKResident" -> "false"),
+          routes.MultipleDisposalsTriageController.checkYourAnswers()
+        )
+      }
+
+      "user has already answered were uk resident section and " +
+        "redirect to dummy page when user re-selected different option" in {
+        val answers = sample[IncompleteMultipleDisposalsAnswers]
+          .copy(wasAUKResident = Some(true), countryOfResidence = Some(Country.uk))
+
+        val (session, journey) = sessionDataWithStartingNewDraftReturn(answers)
+
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+          mockStoreSession(
+            session.copy(journeyStatus = Some(
+              journey.copy(
+                newReturnTriageAnswers = Left(
+                  answers.copy(
+                    wasAUKResident     = Some(false),
+                    countryOfResidence = None
+                  )
+                )
+              )
+            )
+            )
+          )(Right(()))
+        }
+
+        checkIsRedirect(
+          performAction("multipleDisposalsWereYouAUKResident" -> "false"),
+          routes.MultipleDisposalsTriageController.checkYourAnswers()
+        )
+      }
+
+      "user has not answered were uk resident section and " +
+        "submit request without selecting one of wereUkResident options" in {
+        val answers = IncompleteMultipleDisposalsAnswers.empty.copy(
+          individualUserType = Some(Self),
+          numberOfProperties = Some(2)
+        )
+        val (session, _) = sessionDataWithStartingNewDraftReturn(answers)
+
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session)
+        }
+
+        val result = performAction()
+        status(result) shouldBe BAD_REQUEST
```

Comment:
you could use `checkPageIsDisplayed` here as well and assert on the error shown - if you look at a form error spec elsewhere you should see an example